torch/csrc/jit/passes/autocast.cpp

```diff
@@ -24,11 +24,13 @@ bool autocast_enabled = true;
 struct AutocastContext {
   bool gpu_enabled = false;
   bool cpu_enabled = false;
+  bool xla_enabled = false;
```

Comment:
cc @davidberard98 - do you mind reviewing the JIT changes in this file? I'm not too familiar with them.

Reply:
looks good other than the two other comments (static runtime & bc-breaking)